### ISSUE TITLE
REFPROP-manager.hpp glitch in Windows

### ISSUE THREAD

**Issue by bibiloi1** [usnistgov/REFPROP-wrappers#330]
REFPROP-manager,hpp has a glitch when building under Windows (Windows 10 with CMake 3.18 and using Visual Studio 2019 as generator)
Line 88 reads:
     auto load_method = AbstractSharedLibraryWrapper::load_method::LOAD_LIBRARY_PRISTINE;

but LOAD_LIBRARY_PRISTINE throws in windows.h. Under windows use LOAD_LIBRARY instead.
Best regards,
Felix Oltean
Gas & Oil Know-How SRL, Medias, Romania

**Comment by ianhbell** [comment]
I moved the issue to the appropriate repository.

**Comment by ianhbell** [comment]
Yes, windows and linux are different.  Most of the testing has been done in linux, but it works in windows too, you just can't use that option,  I should probably update the README to make this point more clear.
